modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -291,21 +341,15 @@ export default class Ecdsa {
    * @param {XShare} xShare Private xShare of signer
    * @param {YShare} yShare YShare of the other participant involved in
    * this signing operation
-   * @param challenge
+   * @param {SerializedNtilde} challenge
    * @returns {KeyCombined} The new XShare and YShares with the amended
    * challenge values
    */
-  async appendChallenge(
-    xShare: XShare,
-    yShare: YShare,
-    challenge?: DeserializedNtilde
```

Comment:
It was odd to accept the `DeserializedNtilde` as input instead of a `SerializedNtilde`. This was likely due to the circular dependency pointed out above.